cms/templates/library.html

```diff
@@ -65,10 +67,12 @@ <h5 class="title">${_("Library ID")}</h5>
                         </p>
                     </div>
                 </div>
+                % if can_edit:
                 <div class="bit">
                     <h3 class="title-3">${_("Adding content components")}</h3>
                     <p>${_("You can add components to the library. Help text here.")}</p>
```

Comment:
There is a separate PR for this, in review: https://github.com/edx/edx-platform/pull/6426